packages/eslint-plugin/docs/rules/ban-types.md

```diff
@@ -31,17 +31,34 @@ class Foo<F = string> extends Bar<string> implements Baz<string> {
 
 ## Options
 
-The banned type can either be a type name literal (`Foo`), a type name with generic parameter instantiations(s) (`Foo<Bar>`), or the empty object literal (`{}`).
+```ts
+type Options = {
+  types: {
+    [typeName: string]:
+      | string
+      | {
+          message: string;
+          fixWith?: string;
+        };
+  };
+};
+```
+
+The rule accepts a single object as options, with the key `types`.
 
-```CJSON
+- The keys should match the types you want to ban. The type can either be a type name literal (`Foo`), a type name with generic parameter instantiations(s) (`Foo<Bar>`), or the empty object literal (`{}`).
```

Comment:
nit.
```suggestion
- The keys should match the types you want to ban. The type can either be a type name literal (`Foo`), a type name with generic parameter instantiation(s) (`Foo<Bar>`), or the empty object literal (`{}`).
```

Reply:
There is also one inconsistency, this should be align with next line, pick plural or singular form
> The keys should match the types
> The value can be an